core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -3202,6 +3202,10 @@ popup.warning.openOfferWithInvalidMakerFeeTx=The maker fee transaction for offer
   Please go to \"Settings/Network info\" and do a SPV resync.\n\
   For further help please contact the Bisq support channel at the Bisq Matrix Space.
 
+popup.warning.diskSpace=Your free disk space is critically low.\n\
```

Comment:
IMO the error message is not user-friendly. What do you think about "You have less than 2 GB free disk space left. Please free up some disk space to continue running Bisq."